plugins/inputs/logfile/logfile.go

```diff
@@ -394,6 +398,26 @@ func (t *LogFile) cleanUpStoppedTailerSrc() {
 	}
 }
 
+func (t *LogFile) unsetRetentionsForDuplicateLogGroups() error {
```

Comment:
I _think_ I understand what this does, but can you add tests for this?

Reply:
Changed the function to fail based on recent discussions. Also added test, thanks